### PR TITLE
docs: revised the drag and drop feature page

### PR DESCRIPTION
**Pull request by bandantonio**
#### Description of Change

Revised the drag and drop feature page, fixed sample code, added animated image as an example

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Revised the drag and drop feature page with correct sample code and animated image as an example

### CODE REVIEW

File: docs/tutorial/native-file-drag-drop.md [hunk 1]
```diff
@@ -1,29 +1,44 @@
...
+To implement this feature in your app, you need to call
+[`webContents.startDrag(item)`](../api/web-contents.md#contentsstartdragitem)
 API in response to the `ondragstart` event.
 
-In your renderer process, handle the `ondragstart` event and forward the
-information to your main process.
+## Example
+
+Assuming you have a working Electron application from the
+[Quick Start Guide](quick-start.md), add the following lines to the
```
Comment (ckerr) [inline docs/tutorial/native-file-drag-drop.md:18]: Linking back to quick-start is a nice edition; someone landing on this page will be able to walk back to a starting point & be able to make running code.

I can imagine this preface becoming repetitive if we add it to more and more code samples, so I golfed it slightly:

```suggestion
Starting with a working application from the
[Quick Start Guide](quick-start.md), add the following lines to the
```

File: docs/tutorial/native-file-drag-drop.md [hunk 2]
```diff
@@ -1,29 +1,44 @@
...
-    ipcRenderer.send('ondragstart', '/path/to/item')
-  }
-</script>
+<a href="#" id="drag">Drag me</a>
+<script src="renderer.js"></script>
 ```
 
-Then, in the main process, augment the event with a path to the file that is
-being dragged and an icon.
+and add the `renderer.js` file:
```
Comment (ckerr) [inline docs/tutorial/native-file-drag-drop.md:26]: first-app.md points users to electron-quick-start which already has a placeholder `renderer.js`

```suggestion
and add the following lines to the `renderer.js` file:
```

**Comment by bandantonio** [comment]
@ckerr These are great comments, thank you!

**Review by ckerr — APPROVED** [review]

File: docs/tutorial/native-file-drag-drop.md [hunk 3]
```diff
@@ -1,29 +1,44 @@
...
+## Overview
+
 Certain kinds of applications that manipulate files might want to support
 the operating system's native file drag & drop feature. Dragging files into
 web content is common and supported by many websites. Electron additionally
 supports dragging files and content out from web content into the operating
 system's world.
 
-To implement this feature in your app, you need to call `webContents.startDrag(item)`
+To implement this feature in your app, you need to call
```
Comment (erickzhao) [inline docs/tutorial/native-file-drag-drop.md:11]:
```suggestion
To implement this feature in your app, you need to call the
```

File: docs/tutorial/native-file-drag-drop.md [hunk 4]
```diff
@@ -35,3 +50,8 @@ ipcMain.on('ondragstart', (event, filePath) => {
   })
 })
 ```
+
+After launching the Electron application, try to drag-and-drop the item.
```
Comment (erickzhao) [inline docs/tutorial/native-file-drag-drop.md:54]:
```suggestion
After launching the Electron application, try dragging and dropping the item from the app's
BrowserWindow onto your desktop.
```

**Review by erickzhao — APPROVED** [review]
LGTM! Two minor proofread changes.

**Pull request merged**